[2.13][ACM-17851] Updated advanced configuration deployed components for Installer (#7504)
open-cluster-management/rhacm-docs: clusters/install_upgrade/adv_config_install.adoc

|===
| Name | Description | Enabled
| assisted-service | Installs {ocp-short} with minimal infrastructure prerequisites and comprehensive pre-flight validations | True

> | cluster-api-preview | Provides capabilities to declaratively handle the Cluster API lifecycle from within a managment cluster | False

Reviewer: Only line 15 and 16 needed here.

> | cluster-api-preview | Provides capabilities to declaratively handle the Cluster API lifecycle from within a managment cluster | False

Reviewer 2: optional: "declaratively" might be a little hard for translation. Vale tends to flag long adverbs like these, lol 

```suggestion
| cluster-api-preview | Provides capabilities to handle the Cluster API lifecycle from within a management cluster | False
```

Reviewer 2: If you want to keep it, that's cool, but apply the change to fix the minor spelling error: 

```suggestion
| cluster-api-preview | Provides capabilities to declaratively handle the Cluster API lifecycle from within a management cluster | False
```

> | cluster-api-provider-aws-preview | Provides declarative, Kubernetes-style APIs for cluster creation, configuration, and management | False

Reviewer 2: If "declarative" is a term or "type" of API users would be familiar with, keep it! otherwise, I removed it here like I did earlier:

```suggestion
| cluster-api-provider-aws-preview | Provides Kubernetes-style APIs for cluster creation, configuration, and management | False
```

| hive | Provisions and performs initial configuration of {ocp-short} clusters | True
| hypershift | Hosts {ocp-short} control planes at scale with cost and time efficiency, and cross-cloud portability | True
| hypershift-local-hosting | Enables local hosting capabilities for within the local cluster environment | True
| image-based-install-operator | Provides site configuration to {sno} clusters to complete installation | False
| local-cluster | Enables the import and self-management of the local hub cluster where the {mce-short} is deployed | True

> | managedserviceacccount | Synchronizes service accounts to managed clusters, and collects tokens as secret resources to give back to the hub cluster | True

Reviewer: Already reviewed in a previous pr.
